core/shared/platform/windows/platform_internal.h

```diff
@@ -183,9 +183,6 @@ typedef uint32_t os_raw_file_handle;
 // implementation of vprintf on debug builds so output from WASI libc is sent to
 // the debugger and not lost completely.
 #if !defined(BH_VPRINTF) && !defined(NDEBUG) && WINAPI_PARTITION_DESKTOP == 0
-int
-uwp_print_to_debugger(const char *format, va_list ap);
```

Comment:
Not sure why remove the declaration? Will it cause compilation warning when compiling  posix.c?

Reply:
There's already a declaration in `platform_common.h`, [here](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/shared/platform/include/platform_common.h#L55-L62). So this one is redundant. `posix.c` doesn't use `BH_VPRINTF` itself but it includes `platform_common.h` indirectly anyway.